packages/tools/src/utilities/planar/filterAnnotationsWithinSlice.ts

```diff
@@ -41,6 +41,9 @@ export default function filterAnnotationsWithinSlice(
     const data = annotation.data;
     const point = data.handles.points[0];
 
+    if (!annotation.isVisible) {
```

Comment:
@JamesAPetts  is this accurate? as far as planarFreehand is concerned!

Reply:
Same as above.